Add SOE, SPEED, ODOMETER, COOLANT_T, TYRE_T* for Ora Good Cat

spicydog opened the pull request:
Add more parameters for Ora Good Cat

Parameters: 
SOC: State of Charge
SOE: State of Energy
SPEED: Vehicle Speed
ODOMETER: Odometer (Mileage)
COOLANT_TMP: Coolant Temperature
TYRE_P_FL: Tyre Presure Front-Left
TYRE_P_FR: Tyre Presure Front-Right
TYRE_P_RL: Tyre Presure Rear-Left
TYRE_P_RR: Tyre Presure Rear-Right
TYRE_T_FL: Tyre Temperature Front-Left
TYRE_T_FR: Tyre Temperature Front-Right
TYRE_T_RL: Tyre Temperature Rear-Left
TYRE_T_RR: Tyre Temperature Rear-Right

spicydog committed:
Add SOE, SPEED, ODOMETER, COOLANT_T, TYRE_T* to gwm.json

SOC: State of Charge
SOE: State of Energy
SPEED: Vehicle Speed
ODOMETER: Odometer (Mileage)
COOLANT_T: Coolant Temperature
TYRE_P_FL: Tyre Presure Front-Left
TYRE_P_FR: Tyre Presure Front-Right
TYRE_P_RL: Tyre Presure Rear-Left
TYRE_P_RR: Tyre Presure Rear-Right
TYRE_T_FL: Tyre Temperature Front-Left
TYRE_T_FR: Tyre Temperature Front-Right
TYRE_T_RL: Tyre Temperature Rear-Left
TYRE_T_RR: Tyre Temperature Rear-Right

spicydog commented:
Hi @meatpiHQ could you review the pull request or do you need any update?

meatpiHQ commented:
@spicydog you need to add the new parameters in  [schema.json](https://github.com/meatpiHQ/wican-fw/blob/ea8c44b1f9a39ec1660f1af044557f7eb2d2edd1/.vehicle_profiles/schema.json) and change COOLANT_T to COOLANT_TMP

spicydog committed:
Change COOLANT_T to COOLANT_TMP for GWM

spicydog committed:
Update schema.json add Type Temperature

TYRE_T_FL - Tyre Temperature Front Left
TYRE_T_FR - Tyre Temperature Front Right
TYRE_T_RL - Tyre Temperature Rear Left
TYRE_T_RR - Tyre Temperature Rear Right

spicydog commented:
@meatpiHQ I have updated as commented. Please review. Thank you.

jay-oswald commented:
@spicydog you have added to `params.json`, which @meatpiHQ said to do, but thats wrong. They need to go into `params.json`. You have also added `SEO` which wasn't in the list either.

jay-oswald commented:
Sorry, just realised I also made a typo just then @spicydog, but just confirming `params.json` is the correct file to update, `schema.json` is the wrong one, `schema.json` gets automatically built based on `params.json`

spicydog committed:
Revert "Update schema.json add Type Temperature"

This reverts commit 6f754b5d63187a01ea4b2b6f5084cf1f157075c3.

spicydog committed:
Merge branch 'meatpiHQ:main' into patch-1

spicydog committed:
Change SOE to HV_CAPACITY_KWH for GWM

spicydog committed:
Add Tyre temperature to Params

TYRE_T_FL: Tyre Temperature Front Left
TYRE_T_FR: Tyre Temperature Front Right
TYRE_T_RL: Tyre Temperature Rear Left
TYRE_T_RR: Tyre Temperature Rear Right

spicydog commented:
@meatpiHQ @jay-oswald Hi, I have fix as the comments and also change SOE to HV_CAPACITY_KWH which already exists.
Pelase review again. Thank you.

Off-Grid-Garage commented:
> Add more parameters for Ora Good Cat
> 
> Parameters:
SOC: State of Charge
SOE: State of Energy
SPEED: Vehicle Speed
ODOMETER: Odometer (Mileage)
COOLANT_TMP: Coolant Temperature
TYRE_P_FL: Tyre Presure Front-Left
TYRE_P_FR: Tyre Presure Front-Right
TYRE_P_RL: Tyre Presure Rear-Left
TYRE_P_RR: Tyre Presure Rear-Right
TYRE_T_FL: Tyre Temperature Front-Left
TYRE_T_FR: Tyre Temperature Front-Right
TYRE_T_RL: Tyre Temperature Rear-Left
TYRE_T_RR: Tyre Temperature Rear-Right

There are also other parameters like min/max cell voltage and min/max temperature and of course SOH which I would really like to see in HA. Is there a procedure to follow how to get this additional data into HA? I'm happy to help where I can, but unfortunately, I don't know much about the process of adding this to the project and request.

DataBitz commented:
> > Add more parameters for Ora Good Cat
> > 
> > Parameters:
> SOC: State of Charge
> SOE: State of Energy
> SPEED: Vehicle Speed
> ODOMETER: Odometer (Mileage)
> COOLANT_TMP: Coolant Temperature
> TYRE_P_FL: Tyre Presure Front-Left
> TYRE_P_FR: Tyre Presure Front-Right
> TYRE_P_RL: Tyre Presure Rear-Left
> TYRE_P_RR: Tyre Presure Rear-Right
> TYRE_T_FL: Tyre Temperature Front-Left
> TYRE_T_FR: Tyre Temperature Front-Right
> TYRE_T_RL: Tyre Temperature Rear-Left
> TYRE_T_RR: Tyre Temperature Rear-Right
> 
> There are also other parameters like min/max cell voltage and min/max temperature and of course SOH which I would really like to see in HA. Is there a procedure to follow how to get this additional data into HA? I'm happy to help where I can, but unfortunately, I don't know much about the process of adding this to the project and request.
> 

I'm working on some new sensors, SOH is almost ready. Once we have some other sensors people want, I can submit them.
Maybe add to the discussion here of there is something you want to see?
https://github.com/meatpiHQ/wican-fw/discussions/69#discussioncomment-17458325
